### ISSUE TITLE
Schema Core API still uses hapikey

### ISSUE THREAD

**Issue by anka** [HubSpot/hubspot-api-python#132]
Using the Core API of `hubspot.crm.schemas` does not work in version 6.0.0 since it sill uses the`hapikey` for authentication. 

The following code using a token of a private app:

```python
client = hubspot.HubSpot(access_token=os.environ['HUBSPOT_API_TOKEN'])
api = client.crm.schemas.core_api
api.create(object_schema_egg=object_schema)
```

results in

```
HTTP response body: {"status":"error","message":"Authentication credentials not found. 
This API supports both API Key and OAuth 2.0 authentication and you can find more 
details at https://developers.hubspot.com/docs/methods/auth/oauth-overview",
"category":"INVALID_AUTHENTICATION"}
```

Please support `oauth2` authentication for all schema APIs in the library too. See pull request #130.

**Comment by javieralesojeda** [comment]
Thanks for pointing this out @anka It would be great to get this resolved as Private Apps are the only way to use Hubspot's API now. I have been following this thread for a couple of weeks but doesn't seem to get a resolution.

**Comment by anka** [comment]
@javieralesojeda you are welcome. You can use our fix in our private repo, which is based on the current master branch of the library. 

In your`requirements.txt`

```
hubspot-api-client @ git+https://github.com/jademind/hubspot-api-python@a9b4f85c9bd63ab7bc819b6e9fbf56b66acb8740
```

**Comment by javieralesojeda** [comment]
Thanks a lot @anka

**Comment by cdelacombaz** [comment]
This has been fixed on version 7.0.0
